Fix incorrect code returned by ANSI.erase (#4022)
ponylang/ponyc: .release-notes/4022.md

> ## Change ANSI.escape to return the correct escape code
> 
> The erase function states that it erases the line to the left of the cursor, but the escape code it was returning erased to the right of the cursor instead.  
> This fix makes it so that it correctly erases the line to the left of the cursor.

Reviewer: ```suggestion
## Fix incorrect code returned by `ANSI.erase`

`erase` was intended to erase all characters to the left of the cursor but was instead returning the code to erase all characters to the right.
```
